modules/admin_manual/pages/configuration/server/occ_command.adoc

```diff
@@ -226,7 +226,9 @@ include::./app_commands/data_explorer_commands.adoc[leveloffset=+2]
 
 include::./app_commands/ldap_integration_commands.adoc[leveloffset=+2]
 
-include::./app_commands/market_commands.adoc[leveloffset=+2]
+include::./occ_commands/app_commands/files_lifecycle.adoc[leveloffset=+2]
+
+include::./occ_commands/app_commands/ldap_integration_commands.adoc[leveloffset=+2]
```

Comment:
This looks odd. The master PR has all the `include` mentioning path `occ_commands/app_commands`

But lines around here in 10.6 only have `app_commands`

@mmattel @EParzefall ?

Reply:
@phil-davis Yikes, sorry, for the premature merge. I didn't notice something was off. A problem for @mmattel to look into.

Reply:
correctly identified, but no problem. we have reorganized files in their location, but this was not done for 10.6. with this pr, some additional files were created and with that the necessary folders. means it renders correct. yes it may look a bit strange, but we soon will phase out 10.6 because of 10.8. and then, the organisation is harmonized over master/10.8/10.7 :smile: